development/php-stan/Makefile

```
@@ -0,0 +1,45 @@
+delete-stub-wordpress:
+	@if [ -f .cache/stubs/wordpress.stub ]; then \
+  		rm .cache/stubs/wordpress.stub; \
+  	fi
+
+delete-stub-bbpress:
+	@if [ -f .cache/stubs/bbpress.stub ]; then \
+  		rm .cache/stubs/bbpress.stub; \
+  	fi
+
+.PHONY: \
+	download_wordpress \
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)